### DISCUSSION TITLE
Blurry screen dock when maximizing and background overlap seen

### DISCUSSION THREAD

**Discussion by Ms-Ajith** [chriskiehl/Gooey#843] [General]
I found some problems in my created GUI. There is no errors, but some glitches found in GUI. Attaching the screenshots in the word document.

1. When trying to maximise the gui, found the problem.
2. After the successful completion, under the restart tab comes the cancel tab. 
Is there a way to avoid theese? Or are these known issues? Please let me know.
[issues.odt](https://github.com/chriskiehl/Gooey/files/9495813/issues.odt)

**Comment by chriskiehl** [comment]
Unfortunately, I don't understand your question. Further, without you filling out the issue template, I have no idea on which platform you're even experiencing an issue

**Comment by Ms-Ajith** [comment]
from gooey import Gooey, GooeyParser
import argparse
import logging
import sys

# running = True
LOGGER = logging.getLogger("dev")

@Gooey(optional_cols=2,program_name="TOOL")
def main():
    settings_msg = 'Initial GUI for input checker '
    parser = GooeyParser(description=settings_msg)
    parser.add_argument('--verbose', help='be verbose', dest='verbose',
                        action='store_true', default=False)

    subs = parser.add_subparsers(help='commands', dest='command')

    curl_parser = subs.add_parser(
        'curl', help='curl is a tool to transfer data from or to a server')
    curl_parser.add_argument('Path',
                             help='URL to the remote server',
                             type=str, widget='FileChooser')

    parser.parse_args()

if __name__ == '__main__':
    main()

1 - Is it possible to change the label of start and Cancel button which is created automatically.
2. After giving some input, program is successfully completed and shows the result with three Tabs in the bottom 'Edit' 'Restart' 'Close. I could see behind the Restart button, a glimpse of Cancel button from the first screen is visible. Also When I try to enlarge the output window the buttons have glitch problem. The word file is having the screenshots, with my problems. Please have a look.
Thanks in advance
[Dok1.odt](https://github.com/chriskiehl/Gooey/files/9505221/Dok1.odt)
